blerpc/src/test/java/com/blerpc/AnnotationMessageConverterTest.java

```diff
@@ -440,9 +440,14 @@ public void deserializeResponse_emptyMessage() throws Exception {
   }
 
   @Test
-  public void deserializeResponse_wrongMessageByteSize() throws Exception {
-    assertError(() -> converter.deserializeResponse(null, TestIntegerMessage.getDefaultInstance(), new byte[10]),
-        "Declared size 4 of message TestIntegerMessage is not equal to device response size 10");
+  public void deserializeResponse_responseByteSizeLessThatExpected() throws Exception {
+    assertError(() -> converter.deserializeResponse(null, TestIntegerMessage.getDefaultInstance(), new byte[3]),
+        "Declared size 4 of message TestIntegerMessage is bigger, than device response size 3");
+  }
+
+  @Test
+  public void deserializeResponse_responseByteSizeBiggerThatExpected() throws Exception {
+    converter.deserializeResponse(null, TestIntegerMessage.getDefaultInstance(), new byte[10]);
```

Comment:
Тест не читаемый. Ты вызвал метод, ок, я не вижу, чтоб что-то тут тестировалось.

Reply:
Я хотел протестировать, что не будет ошибки, если количество байт больше, чем ожидалось. Тогда в тесте укажу, что проверка на то, что нет ошибки. В `BleRpcChannelTest` мы делали это с помощью комментария:

```
// If test fail it will throw RuntimeException.
```

Предлагаю сделать тут также, что думаешь?